packages/web_benchmarks/lib/src/recorder.dart

```diff
@@ -604,10 +602,10 @@ class Timeseries {
 
   /// The number of frames ignored as warm-up frames, used only
   /// when [useCustomWarmUp] is true.
-  int _warmUpFrameCount;
+  int? _warmUpFrameCount;
 
   /// The number of frames ignored as warm-up frames.
-  int get warmUpFrameCount =>
+  int? get warmUpFrameCount =>
```

Comment:
If `_warmUpFrameCount` is expected to be non-null whenever `useCustomWarmUp` is true, then shouldn't this return be non-nullable, and `_warmUpFrameCount` be force-unwrapped in the next line?